docs/security-considerations.rst

```diff
@@ -22,7 +22,10 @@ you should be more careful.
 This section will list some pitfalls and general security recommendations but
 can, of course, never be complete. Also, keep in mind that even if your
 smart contract code is bug-free, the compiler or the platform itself might
-have a bug.
+have a bug. All known security-relevant bugs of the compiler can be found in the
```

Comment:
Also, I wouldn't say "all".  Any discrepancy between expectation and implementation is a bug, and we don't know people's expectations.